fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/SessionVariableDef.java

```diff
@@ -12,35 +12,31 @@
 
 /**
  * Adds a session variable to the database
-
- *
  */
 public class SessionVariableDef extends BaseObject {
-    
+
     public SessionVariableDef(String schemaName, String variableName, int version) {
         super(schemaName, variableName, DatabaseObjectType.VARIABLE, version);
     }
-    
 
-    /* (non-Javadoc)
-     * @see com.ibm.fhir.database.utils.model.IDatabaseObject#apply(com.ibm.fhir.database.utils.api.IDatabaseAdapter)
-     */
     @Override
     public void apply(IDatabaseAdapter target) {
         target.createIntVariable(getSchemaName(), getObjectName());
     }
 
-    /* (non-Javadoc)
-     * @see com.ibm.fhir.database.utils.model.IDatabaseObject#drop(com.ibm.fhir.database.utils.api.IDatabaseAdapter)
-     */
+    @Override
+    public void apply(Integer priorVersion, IDatabaseAdapter target) {
+        target.createIntVariable(getSchemaName(), getObjectName());
```

Comment:
we should not have logic here to apply a new version.  Only the first time. it's like a sequence, very integral to the overall operation.